lib/instance-validator.js

```diff
@@ -108,7 +108,7 @@ InstanceValidator.prototype._builtinValidators = function() {
   var validators = [];
   Utils._.forIn(this.modelInstance.rawAttributes, function(rawAttribute, field) {
     if (self.options.skip.indexOf(field) >= 0) {
-      return;
+      return false;
```

Comment:
Is the iteration supposed to be exited early though? `skip` might match the first field, should still validate all the other fields.